blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubPipelineCreateRequest.java

```diff
@@ -179,8 +186,10 @@ public boolean meetsIndexingCriteria() {
                         }
                     });
                 }else {
+                    gitHubSCMNavigator.setPattern(".*");
```

Comment:
sure - not really worried about optimisation here as not exactly a warm code path!

With testing we can rework this, but yeah as you said, laborious to test!